spec/features/article_admin_feature_spec.rb

```diff
@@ -40,11 +36,7 @@
 
   scenario "admin user deletes article" do
     @article = create(:article)
-    visit articles_path
-    click_link "New Article"
-    fill_in :article_title, with: @article.title
-    fill_in :article_body, with: @article.body
-    click_button "Submit"
+    visit article_path(@article)
     click_link "Delete Article"
     page.driver.browser.switch_to.alert.accept
```

Comment:
I tried using the following Capybara method: 
```
page.accept_alert "Are you sure you want to delete this article?" do
  click_button('OK')
end
```
but I keep receiving the same error due to the selenium-webdriver gem: 
```
Selenium::WebDriver::Error::UnhandledAlertError:
       Unexpected modal dialog (text: Are you sure you want to delete this article?): "Are you sure you want to delete this article?"
```